federatedscope/core/fed_runner.py

```diff
@@ -170,13 +237,21 @@ def is_broadcast(msg):
                 self.client.join_in()
                 self.client.run()
 
-    def _setup_server(self):
+    def _setup_server(self, device_info=None, client_info=None):
         """
         Set up the server
         """
         self.server_id = 0
         if self.mode == 'standalone':
-            if self.server_id in self.data:
+            if self.cfg.federate.merge_test_data:
+                from federatedscope.core.auxiliaries.data_builder import \
```

Comment:
does this module involves backend-related dependence? if it is not the case, no need to put import here

Reply:
I have moved the import statement to the top of the file